rust/connlib/tunnel/src/lib.rs

```diff
@@ -214,8 +214,20 @@ where
             }
         }
 
-        let relay_socket_v4 = Socket::bind((IpAddr::from(Ipv4Addr::UNSPECIFIED), 0));
-        let relay_socket_v6 = Socket::bind((IpAddr::from(Ipv6Addr::UNSPECIFIED), 0));
+        // TODO: We need to make this dynamic and cannot fail on missing IPv4 or IPv6.
```

Comment:
@thomaseizinger this is just to test CI right? The problem is we don't know which interface can reach the relays (and it can change over time). This could even happen in CI